tests/webapp/api/test_jobs_api.py

```diff
@@ -154,6 +157,49 @@ def test_job_list_in_filter(webapp, eleven_jobs_stored, jm):
     assert len(resp['results']) == 2
 
 
+def test_job_list_excluded(webapp, eleven_jobs_stored, sample_data, test_project):
+    """
+    retrieve a job list of only jobs excluded by specified profile
+    """
+    job = sample_data.job_data[1]
+    platform = job["job"]["machine_platform"]["platform"]
+    arch = job["job"]["machine_platform"]["architecture"]
+
+    user = User.objects.create(username="testuser2",
+                               email='foo2@bar.com',
+                               is_staff=True)
+
+    exclusion_profile = ExclusionProfile(
+        name="exprof",
+        is_default=False,
+        author=user
+    )
+    exclusion_profile.save()
+
+    job_exclusion = JobExclusion(
+        name="jobex",
+        info={
+            "platforms": ["{} ({})".format(platform, arch)],
+            "repos": [test_project],
+            "option_collections": ["opt"],
+            "option_collection_hashes": ["102210fe594ee9b33d82058545b1ed14f4c8206e"],
+            "job_types": ["B2G Emulator Image Build (B)"]},
+        author=user,
+    )
+    job_exclusion.save()
+    job_exclusion.profiles.add(exclusion_profile)
+
+    exclusion_profile.save()
+
+    url = reverse("jobs-list",
+                  kwargs={"project": test_project})
+    final_url = url + ("?exclusion_profile=exprof&visibility=excluded")
+
+    resp = webapp.get(final_url).json
```

Comment:
I would just do:
```py
webapp.get(reverse('jobs-list', kwargs={"project": test_project}) + ("?exclusion_profile=exprof&visibility=excluded")
```

here. No need to have a bunch of intermediary variables, I don't think.